src/ucp/core/ucp_worker.c

```diff
@@ -300,6 +304,9 @@ ucp_worker_iface_error_handler(void *arg, uct_ep_h uct_ep, ucs_status_t status)
     if (ucp_ep_errh_iter != kh_end(&worker->ep_errh_hash)) {
         err_handler = kh_val(&worker->ep_errh_hash, ucp_ep_errh_iter);
         err_handler.cb(err_handler.arg, ucp_ep, status);
+    } else {
+        ucs_error("Error %s was not handled for ep %p",
```

Comment:
i don't think we should add this print, let the transport print the error as today